test/expect/TestScript.test_constant_pooling.expect

```diff
@@ -2,24 +2,24 @@ graph(%cond : Tensor):
   %1 : int[] = prim::Constant[value=[1]]()
   %2 : int[] = prim::Constant[value=[0]]()
   %3 : int = prim::Constant[value=3]()
-  %4 : Float(2) = prim::Constant[value= 4  4 [ Variable[CPUFloatType]{2} ]]()
-  %5 : Float(2) = prim::Constant[value= 1  1 [ Variable[CPUFloatType]{2} ]]()
+  %4 : Double(2) = prim::Constant[value= 4  4 [ Variable[CPUDoubleType]{2} ]]()
```

Comment:
This is now consistent with the default dtype

Reply:
is this what we want?  It means that tracing matches eager (i.e., respects the default dtype), but doesn't match script (which doesn't respect the default dtype).  Seems like a reasonable decision to me, but worth checking with @zdevito.

Reply:
I think this is fine.  I don't get the impression dtype defaults are getting changed much apart from debugging.